pages/common/python.md

```diff
@@ -23,6 +23,10 @@
 
 `python -m {{module}} {{arguments}}`
 
+- Install a package:
```

Comment:
I see. In that case, I guess it's ok.

Let's make this a little more explicit though, just to be on the safe side:
```suggestion
- Install a package using pip:
```

I'm not particularly experienced with Python, so a 2nd opinion from someone who is would be appreciated.

Reply:
That seems all right, committed! :)